### PR TITLE
fixed missing dependencies: PERL5LIB

### PR DESCRIPTION
**Pull request by qknight**
fix for https://github.com/nixos/hydra/issues/208 issue

### CODE REVIEW

File: release.nix [hunk 1]
```diff
@@ -158,6 +159,7 @@ in rec {
 
         for i in $out/bin/*; do
             wrapProgram $i \
+                --set PERL5LIB "${stdenv.lib.makePerlPath perlPaths}" \
```
Comment (edolstra) [inline release.nix]: This is not a good idea since it makes Hydra startup much slower.

File: release.nix [hunk 2]
```diff
@@ -121,12 +118,16 @@ in rec {
             SetScalar
             Starman
             SysHostnameLong
-            TestMore
             TextDiff
             TextTable
             XMLSimple
             nix git
+            BHooksEndOfScope
```
Comment (edolstra) [inline release.nix]: Where do we use this?

Comment (qknight) [inline release.nix]: not sure, but the error said:

<pre>
Could not find a suitable B::Hooks::EndOfScope implementation: Can't locate Variable/Magic.pm:   Permission denied at /nix/store/7x2a4nk6dv8wkx9bkmhl96kbmqzfmb2h-hydra-perl-deps/lib/perl5/site_perl/5.20.1/Module/Runtime.pm line 317.
BEGIN failed--compilation aborted at /nix/store/7x2a4nk6dv8wkx9bkmhl96kbmqzfmb2h-hydra-perl-deps/lib/perl5/site_perl/5.20.1/B/Hooks/EndOfScope/XS.pm line 17.
Compilation failed in require at /nix/store/7x2a4nk6dv8wkx9bkmhl96kbmqzfmb2h-hydra-perl-deps/lib/perl5/site_perl/5.20.1/Module/Runtime.pm line 317.
Can't locate Tie/StdHash.pm:   Permission denied at /nix/store/698jpw5xd0pfc332xm30azn9zhrb0546-perl-5.20.1/lib/perl5/5.20.1/base.pm line 99.
        ...propagated at /nix/store/698jpw5xd0pfc332xm30azn9zhrb0546-perl-5.20.1/lib/perl5/5.20.1/base.pm line 108.
BEGIN failed--compilation aborted at /nix/store/7x2a4nk6dv8wkx9bkmhl96kbmqzfmb2h-hydra-perl-deps/lib/perl5/site_perl/5.20.1/B/Hooks/EndOfScope/PP/FieldHash.pm line 29.
Compilation failed in require at /nix/store/7x2a4nk6dv8wkx9bkmhl96kbmqzfmb2h-hydra-perl-deps/lib/perl5/site_perl/5.20.1/Module/Runtime.pm line 317.
BEGIN failed--compilation aborted at /nix/store/7x2a4nk6dv8wkx9bkmhl96kbmqzfmb2h-hydra-perl-deps/lib/perl5/site_perl/5.20.1/B/Hooks/EndOfScope/PP.pm line 29.
Compilation failed in require at /nix/store/7x2a4nk6dv8wkx9bkmhl96kbmqzfmb2h-hydra-perl-deps/lib/perl5/site_perl/5.20.1/Module/Runtime.pm line 317.
</pre>

File: release.nix (hunk 1 above)
Comment (qknight) [inline release.nix]: ok

**Comment by domenkozar** [comment]
I believe this can be closed now

**Pull request closed without merging**